content/developers.html

```diff
@@ -267,17 +335,16 @@ <h3 class="panel-title">Git Repositories</h3>
 <ul>
 <li><a href="https://gitbox.apache.org/repos/asf/cloudstack.git" target="_blank">Apache CloudStack source code</a></li>
 <li><a href="https://gitbox.apache.org/repos/asf/cloudstack-cloudmonkey.git" target="_blank">Apache CloudStack Cloudmonkey source code</a></li>
-<li><a href="https://github.com/apache/cloudstack-ec2stack" target="_blank">Apache CloudStack EC2stack Inteface</a></li>
-<li><a href="https://github.com/apache/cloudstack-gcestack" target="_blank">Apache CloudStack GCEstack Interface</a></li>
-<li><a href="https://github.com/apache/cloudstack-docs" target="_blank">General Documentation</a></li>
-<li><a href="https://github.com/apache/cloudstack-docs-install" target="_blank">Installation Guide</a></li>
-<li><a href="https://github.com/apache/cloudstack-docs-admin" target="_blank">Administrative Guide</a></li>
-<li><a href="https://github.com/apache/cloudstack-docs-rn" target="_blank">Release Notes</a></li>
```

Comment:
Should we eventually deprecate and remove the link? What do you think @PaulAngus ?

Reply:
yes eventually.  but 4.9 is still relevant to alot of people just now